### PR TITLE
APICLIENT-16: Add featured image

### PR DESCRIPTION
**Pull request by steffenkleinle**

### CODE REVIEW

File: src/endpoints/createEventsEndpoint.js [hunk 1]
```diff
@@ -50,7 +51,14 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<EventModel>> => new
...
-        hash: event.hash
+        hash: event.hash,
+        featuredImage: event.featured_image ? new FeaturedImageModel({
+          description: event.featured_image.description,
+          thumbnail: event.featured_image.thumbnail[0],
+          medium: event.featured_image.medium[0],
+          large: event.featured_image.large[0],
+          full: event.featured_image.full[0]
+        }) : null
       })
```
Comment (michael-markl) [inline src/endpoints/createEventsEndpoint.js:62]: Do we only have a featured image for events?

**Review by michael-markl — APPROVED** [review]

File: src/endpoints/createEventsEndpoint.js (hunk 1 above)
Comment (steffenkleinle) [reply src/endpoints/createEventsEndpoint.js:62]: I think so, at least pages does not have a featured image yet. Maybe the pois, but not sure.

Comment (steffenkleinle) [reply src/endpoints/createEventsEndpoint.js:62]: I checked it, only events right now.

File: src/models/FeaturedImageModel.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  _full: FeaturedImageInstanceType
+
+  constructor (params: {| description: ?string, thumbnail: FeaturedImageInstanceType,
+    medium: FeaturedImageInstanceType, large: FeaturedImageInstanceType, full: FeaturedImageInstanceType |}) {
+    this._description = params.description
+    this._thumbnail = params.thumbnail
+    this._medium = params.medium
+    this._large = params.large
+    this._full = params.full
+  }
```
Comment (maxammann) [inline src/models/FeaturedImageModel.js:23]: I think the get functions are missing here

**Review by maxammann — APPROVED** [review]

File: src/models/FeaturedImageModel.js (hunk 2 above)
Comment (steffenkleinle) [reply src/models/FeaturedImageModel.js:23]: Impossible! :sunglasses:

**Pull request merged**